css/css-backgrounds/border-image-width-should-extend-to-padding.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="author" title="Tyler Wilcock" href="mailto:twilco.o@protonmail.com">
+<link rel="help" href="https://www.w3.org/TR/2017/CR-css-backgrounds-3-20171017/#the-border-image-width">
```

Comment:
Can you use https://drafts.csswg.org/css-backgrounds/#border-image-width ?

Reply:
Done.